src/js/plugin.js

```diff
@@ -76,6 +76,26 @@ const markPluginAsActive = (player, name) => {
 };
 
 /**
+ * Triggers a pair of plugin setup events.
+ *
+ * @private
+ * @param  {Player} player
+ *         A Video.js player instance.
+ *
+ * @param  {Plugin~PluginEventHash} hash
+ *         A plugin event hash.
+ *
+ * @param  {Boolean} [before]
+ *         If true, modifies the events to be "before" events.
```

Comment:
I think it would be good to have an example, as this confused me at first. Maybe `I.E `pluginsetup` -> `beforepluginsetup``